src/unittest/test.cpp

```diff
@@ -255,7 +243,12 @@ bool run_tests()
 	}
 
 	rawstream << "Catch test results: " << std::endl;
-	auto num_catch_tests_failed = Catch::Session().run();
+	Catch::Session session{};
+	auto config = session.configData();
+	config.skipBenchmarks = true;
+	config.allowZeroTests = true;
+	session.useConfigData(config);
+	auto num_catch_tests_failed = session.run();
```

Comment:
This returns an exit code now, not the number of tests failed. Let's not hide that under a false name.

Reply:
indeed, done